src/framework/theme/styles/themes/_mapping.scss

```diff
@@ -1761,4 +1761,76 @@ $eva-mapping: (
   icon-success-color: color-success-default,
   icon-warning-color: color-warning-default,
   icon-danger-color: color-danger-default,
+
+  toggle-height: 1.875rem,
+  toggle-width: 3.125rem,
+  toggle-border-width: 1px,
+  toggle-border-radius: 100px,
+  toggle-border-color: border-basic-color-4,
+  toggle-background-color: background-basic-color-3,
+  toggle-outline-width: outline-width,
+  toggle-outline-color: outline-color,
+  toggle-switcher-size: 1.75rem,
+  toggle-switcher-background-color: background-basic-color-1,
+  toggle-switcher-checkmark-color: color-primary-default,
+
+  toggle-disabled-background-color: background-basic-color-2,
+  toggle-disabled-border-color: border-basic-color-3,
+  toggle-disabled-checkmark-color: background-basic-color-1,
```

Comment:
Everywhere else variables are called `switcher-checkmark`, not just `checkmark`.